spec/views/catalog/show.html.erb_spec.rb

```diff
@@ -23,15 +23,13 @@
       it 'will display one viewer from 2 arks, one is suppressed' do
```

Comment:
These fixtures (6868324, 6773431, 6109323) were used to support the Javascript code which is looking for arks in the marc_record when the iiif manifest path is not indexed. 
Since there is no test for the async call, I'm testing manually the fixtures to see if the viewer is displayed properly. For these 3 fixtures the viewer does not display properly. These are the fixtures that they don't have indexed iiif manifests in the marc_record.